src/main/kotlin/org/partiql/scribe/sql/AstToSql.kt

```diff
@@ -0,0 +1,9 @@
+package org.partiql.scribe.sql
+
+import org.partiql.ast.sql.SqlDialect
+import org.partiql.scribe.ScribeContext
+
+/**
+ * Base class for AST to SQL translators.
+ */
+public abstract class AstToSql(public val context: ScribeContext) : SqlDialect()
```

Comment:
self-review: refactor to wrap PLK's SqlDialect and allow passing a `ScribeContext`, so we can report scribe problems in the AST -> text translation. the corresponding dialects for Redshift, Trino, and Spark have also been renamed